.cirrus.yml

```diff
@@ -59,6 +59,18 @@ validate_task:
         image_name: ${DEBIAN_CACHE_IMAGE_NAME}
     env:
         HOME: "/root"  # default unset, needed by golangci-lint.
+    go_modules_cache: &go_modules_cache
+        fingerprint_script: cat go.sum
+        folder: $GOPATH/pkg/mod
+    go_build_cache: &go_build_cache
+        fingerprint_script: cat go.sum
+        folder: $GOCACHE
+    golangci-lint_cache:
+        folder: /root/.cache/golangci-lint
+        reupload_on_changes: true
+        fingerprint_script:
+            - go version
+            - grep GOLANGCI_LINT_VERSION Makefile | head -1
     script: |
         git remote update
```

Comment:
Note to self: This seems mostly unnecessary, we just want the destination branch.